localstack/aws/protocol/parser.py

```diff
@@ -509,30 +509,54 @@ def __init__(self, service: ServiceModel) -> None:
         self.operation_lookup: DefaultDict[
             str, DefaultDict[Pattern[str], OperationModel]
         ] = defaultdict(lambda: defaultdict())
-        for operation in service.operation_names:
-            operation_model = service.operation_model(operation)
+        # Extract all operation models from the service spec
+        operation_models = [
+            service.operation_model(operation) for operation in service.operation_names
+        ]
+        # Exclude all operation models which do not have "http" metadata
+        filtered_operation_models = filter(lambda model: len(model.http) > 0, operation_models)
```

Comment:
We had the check previously, I just didn't want to add it multiple times.
Initially, I just added it for safety reasons since I thought it might not be safe to assume that the HTTP section is always there...
What's your opinion? Dropping the filter? New specs are usually `rest-json`, so I think it might be safe that upcoming specs won't drop the HTTP section either.

Reply:
:shrug: not sure it's super critical. but if there's case for it yet then i would remove it for now. when something breaks, we'll be aware of the case ;-)